spec/watirspec/elements/td_spec.rb

```diff
@@ -77,4 +77,20 @@
     end
   end
 
+  describe "#sibling_from_header" do
+    it "returns the corresponding sibling cell by text" do
+      local_td = browser.td(text: '1 331').tap(&:exist?)
```

Comment:
because I had a debug point right after it to check on things as I was testing it. :-D
They don't belong in the final code, thanks.